assets/css/admin/onboarding/style.scss

```diff
@@ -0,0 +1,762 @@
+	content: "\f122";
+	font-family: Dashicons;
+	font-size: 14px;
+	font-weight: 400;
+	color: #dedfe0;
+	margin-right: 3px;
+}
+
+.edd-onboarding__testimonial-content > span.big {
+	font-weight: 600;
+	font-size: 15px;
+	font-style: italic;
+}
+
+.edd-onboarding__testimonial-person {
+	display: flex;
+	align-self: flex-end;
+}
+
+.edd-onboarding__testimonial-avatar {
+	vertical-align: middle;
+	width: 50px;
+	height: 50px;
+	border-radius: 50%;
+}
+
+.edd-onboarding__testimonial-info {
+	display: flex;
+	flex-direction: column;
+	margin-left: 10px;
+	margin-top: 0px;
+}
+
+.edd-onboarding__testimonial-info > .testimonial-name {
+	font-weight: 600;
+}
+
+.edd-onboarding__testimonial-info > .testimonial-company {
+	font-size: 10px;
+	font-style: italic;
+}
+
+.edd-onboarding__testimonial-info > .testimonial-stars > span {
+	color: #ffbb38;
+	font-size: 12px;
+	height: 12px;
+	width: 12px;
+}
+
+.edd-onboarding__welcome-screen-get-started {
+	color: white;
+	background: #24c261;
+	padding: 10px 30px;
+	display: inline-block;
+	text-decoration: none;
+	margin-top: 10px;
+	&:hover {
+		color: white;
+		background: #27da6c;
+	}
+}
+
+.edd-onboarding__plugins-list {
+	border-top: 1px solid rgba(#ededed, 0.5);
+	.edd-onboarding__plugins-plugin {
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+		padding: 28px 20px;
+		border-bottom: 1px solid rgba(#ededed, 0.5);
+		border-left: 1px solid rgba(#ededed, 0.5);
+		border-right: 1px solid rgba(#ededed, 0.5);
+		transition: all 0.25s ease-out;
+		.edd-onboarding__plugins-control {
+			width: 100px;
+			text-align: right;
+	   }
+		.edd-onboarding__plugins-details {
+			h3 {
+				margin-top: 0;
+				transition: all 0.25s ease-out;
+		   }
+			p {
+				margin-bottom: 0;
+				transition: all 0.25s ease-out;
+		   }
+	   }
+		&.disabled {
+			background: rgba(#72b281, 0.04);
+			.edd-onboarding__plugins-details {
+				h3 {
+					//color: #72b281;
+			   }
+				p {
+					//color: #72b281;
+			   }
+		   }
+			&:hover {
+				background: rgba(#72b281, 0.04);
+				.edd-onboarding__plugins-details {
+					h3 {
+						color: #141b38;
+				   }
+					p {
+						color: #141b38;
+				   }
+			   }
+		   }
+	   }
+		&:hover {
+			background: rgba(#0c5d95, 0.02);
+			.edd-onboarding__plugins-details {
+				h3 {
+					color: #0c5d95;
+			   }
+				p {
+					color: #0c5d95;
+			   }
+		   }
+	   }
+   }
+}
+.edd-onboarding__single-step-footer {
+	border-top: 1px solid #ededed;
+	padding: 35px 50px 35px 50px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	.edd-onboarding__button-back {
+		color: #8a8e92;
+		text-decoration: none;
+		transition: all 0.2s ease-in-out;
+		&:hover {
+			color: #0c5d95;
+	   }
+   }
+}
+
+.edd-onboarding__close-and-exit-wrapper {
+	text-align: center;
+	a {
+		color: #8a8e92;
+		opacity: 0.5;
+		margin-top: 20px;
+		display: block;
+	}
+}
+
+/* EDD ONBOARDING - Buttons style */
+.wp-core-ui .edd-onboarding .edd-onboarding__button.button,
```

Comment:
I don't know about overriding all of the button styles. I'm wondering if you can get what you want with using the default WP button classes and add `button-large` or `button-hero`.